index.html

```diff
@@ -560,17 +560,59 @@ <h4>$schema</h4>
-               <h4>name</h4>
+               <h4><dfn>title</dfn></h4>
                <p>
-                  It is RECOMMENDED that all JSON Schemas include a <code>name</code> property which provides
-                  a human-readable name that describes the schema.
+                  It is RECOMMENDED that all JSON Schemas include the optional <code>title</code> property as defined in
+                  [[JSON-SCHEMA]].
                </p>
             </section>
             <section class="normative">
-               <h4>description</h4>
+               <h4><dfn>description</dfn></h4>
                <p>
-                  JSON Schemas MAY choose to include an optional <code>description</code> property which provides
-                  a human-readable sentence describing the schema.
+                  It is RECOMMENDED that all JSON Schemas include the optional <code>description</code> property as
+                  defined in [[JSON-SCHEMA]].
+               </p>
+            </section>
+            <section class="normative">
+               <h4>dir</h4>
+               <p>
+                  JSON Schemas MAY choose to include an optional <code>dir</code> property that indicates the
+                  base direction for the values of the [=title=] and [=description=] properties defined in
+                  the JSON Schema. The value of this property MUST be a <a>text-direction</a>. When absent, implementers
+                  MUST assume that the value of this property is "[=text-direction/auto=]".
+               </p>
+              <p>
+                The <dfn>text-directions</dfn> are the following:
+              </p>
+              <dl>
+                <dt>
+                  "<dfn data-dfn-for="text-direction">ltr</dfn>"
+                </dt>
+                <dd>
+                  Left-to-right text.
+                </dd>
+                <dt>
+                  "<dfn data-dfn-for="text-direction">rtl</dfn>"
+                </dt>
+                <dd>
+                  Right-to-left text.
+                </dd>
+                <dt>
+                  "<dfn data-dfn-for="text-direction">auto</dfn>" (default)
+                </dt>
+                <dd>
+                  No explicit directionality.
+                </dd>
+              </dl>
+            </section>
+            <section class="normative">
+               <h4>lang</h4>
+               <p>
+                 JSON Schemas MAY choose to include an optional <code>lang</code> property that indicates the
+                 <a data-cite="ltli/#dfn-language-tag">language tag</a> for the values of the [=title=] and
+                 [=description=] properties defined in the JSON Schema. The value of this field MUST be a
+                  <a data-cite="ltli/#dfn-canonical-unicode-locale-identifier">canonical unicode locale identifier</a>.
+                  When absent, implementers MUST assume that the value of this property is `en-US`.
```

Comment:
I don't agree with this in app-manifest (I think we have an issue open against it?). The value when the language is not present should be `und` ("Undetermined", also known as the "root" locale)

Reply:
I'm not sure what issue/repo you may be referring to. Mind clarifying?

Curious, is that `und` guidance part of the LTLI spec?

I've updated this PR.

Reply:
> Curious, is that und guidance part of the LTLI spec?

Actually, no, it's in [String-Meta](https://www.w3.org/TR/string-meta/#bp_default_fallback) and the [I18N Best Practices for Specs](https://www.w3.org/TR/international-specs/#bp_legacy_fmt_lang_unknown) document. I've created an issue to add it to LTLI.